scss/util/_mixins.scss

```diff
@@ -92,14 +92,15 @@
     height: $weight;
 
     background: $color;
-    box-shadow: $shadow;
 
     content: '';
 
     @for $i from 2 through $bars {
       $offset: ($weight + $spacing) * ($i - 1);
       $shadow: append($shadow, 0 $offset 0 $color, comma);
     }
+
+    box-shadow: $shadow;
```

Comment:
I think it would be more clear to keep `box-shadow` in the property list `background`, and move the logic stuff above, line 75 or 86.